tests/foreman/api/test_host.py

```diff
@@ -610,105 +762,88 @@ def test_positive_delete(self):
-        with self.assertRaises(HTTPError):
+        with pytest.raises(HTTPError):
             host.read()
 
-    @tier1
-    def test_positive_update_name(self):
-        """Update a host with a new name
-
-        :id: a82b606c-d683-44ba-9086-684396ef1c10
-
-        :expectedresults: A host is updated with expected name
-
-        :CaseImportance: Critical
-        """
-        host = entities.Host().create()
-        for new_name in valid_hosts_list():
-            with self.subTest(new_name):
-                host.name = new_name
-                host = host.update(['name'])
-                self.assertEqual(host.name, '{0}.{1}'.format(new_name, host.domain.read().name))
-
-    @tier1
-    def test_positive_update_mac(self):
-        """Update a host with a new MAC address
-
-        :id: 72e3b020-7347-4500-8669-c6ddf6dfd0b6
-
-        :expectedresults: A host is updated with a new MAC address
-
-        :CaseImportance: Critical
-        """
-        host = entities.Host().create()
-        new_mac = gen_mac(multicast=False)
-        host.mac = new_mac
-        host = host.update(['mac'])
-        self.assertEqual(host.mac, new_mac)
-
     @tier2
-    def test_positive_update_domain(self):
-        """Update a host with a new domain
+    def test_positive_create_and_update_domain(self, module_org, module_location):
+        """Create and update a host with a domain
 
         :id: 8ca9f67c-4c11-40f9-b434-4f200bad000f
 
-        :expectedresults: A host is updated with a new domain
+        :expectedresults: A host is created and updated with expected domain
 
         :CaseLevel: Integration
         """
-        host = entities.Host().create()
+        domain = entities.Domain(organization=[module_org], location=[module_location]).create()
+        host = entities.Host(
+            organization=module_org, location=module_location, domain=domain
+        ).create()
+        assert host.domain.read().name == domain.name
+
         new_domain = entities.Domain(
-            location=[host.location], organization=[host.organization]
+            organization=[module_org], location=[module_location]
         ).create()
         host.domain = new_domain
         host = host.update(['domain'])
-        self.assertEqual(host.domain.read().name, new_domain.name)
+        assert host.domain.read().name == new_domain.name
 
     @tier2
-    def test_positive_update_env(self):
-        """Update a host with a new environment
+    def test_positive_create_and_update_env(self, module_org, module_location):
+        """Create and update a host with an environment
 
         :id: 87a08dbf-fd4c-4b6c-bf73-98ab70756fc6
 
-        :expectedresults: A host is updated with a new environment
+        :expectedresults: A host is created and updated with expected environment
 
         :CaseLevel: Integration
         """
-        host = entities.Host().create()
+        env = entities.Environment(organization=[module_org], location=[module_location],).create()
```

Comment:
module_env should be available in global fixtures

Reply:
DONE.